jscomp/bsb/templates/react/webpack.config.js

```diff
@@ -1,13 +1,9 @@
 const path = require('path');
 
 module.exports = {
-  entry: {
-    async: './lib/js/src/async/counterRoot.js',
-    simple: './lib/js/src/simple/simpleRoot.js',
-    interop: './src/interop/interopRoot.js',
-  },
+  entry: './lib/js/src/index.js',
```

Comment:
could we use the in-source option and suffix .bs.js?

Reply:
Probably...? What's the benefit of using that instead?

Reply:
so you don't need remember magic path like `lib/js`, it is also more robust with deletion/addition of files